Elements/RightDescriptionChoiceElement.cs

```diff
@@ -0,0 +1,167 @@
+﻿using System;
+using Mono.Cecil.Cil;
+using MonoDetour;
+using MonoDetour.Cil;
+using MonoDetour.HookGen;
+using MonoMod.Cil;
+using Silksong.ModMenu.Internal;
+using Silksong.ModMenu.Models;
+using Silksong.UnityHelper.Extensions;
+using UnityEngine;
+using UnityEngine.UI;
+using UObject = UnityEngine.Object;
+
+namespace Silksong.ModMenu.Elements;
+
+/// <summary>
+/// Subclass of <see cref="ChoiceElement{T}"/> with a second description object that lies below the choice text.
+/// </summary>
+public class RightDescriptionChoiceElement<T> : ChoiceElement<T>
```

Comment:
I think 'DynamicDescriptionChoiceElement' was a better name, this new name doesn't really convey what makes it different from 'ChoiceElement' (at a cursory glance I'd think it was strictly an alignment change).